ansible/group_vars/all.yml.sample

```diff
@@ -188,3 +188,27 @@ kafka_instances:
   key: kafka
   value: "true"
 kafka_node_label: 'kafka: "true"'
+
+################################################################################
+# CNF profile
+################################################################################
+# PTP interface (precision time protocol)
+# This is the interface used for ptp4l, in an OpenShift cluster this typically is the baremetal interface
+ptp_interface: "eno1"
+# Hugepages
+hugepage_size: "1G"
+hugepage_count "16"
+hugepage_node: "0"
+# NUMA
+# temp workaround for RT kernel bugs that consume too much CPU power: add more isolated cores
+isolated: "1,3,5,7,9-51"
+reserved: "0,2,4,6,8"
+realtime_kernel "true"
+# MachineConfigPool that gets to run more pods
+max_pod_mcp: "worker"
+max_pods: 500
+# NodeSelector and MachineConfigPoolSelector for the PerformanceProfile using realtime kernel 
+# and Hugepages
+perf_mcp_selector: "master: \"\""
```

Comment:
@mkarg75 @mukrishn , I think we should ideally make the entire nodeSelector and machineConfigPoolSelector variables to give the user flexibility. To me, it doesn't make sense to variablize only a part of the variable. So instead of
```
nodeSelector:
    node-role.kubernetes.io/{{ perf_node_selector }}
```

it would become
```
nodeSelector:
   {{ perf_node_selector }}
```

Updating the variables accordingly.

Just my 0.02, as we should probably make this a true variable if we are going to variabilize it, instead of hardcoding part. Also, leaves the entire responsibility and power to the user to use any free-form variable they like. Let me know what you think.

Looks good to me otherwise.

Reply:
I've made those two full variables, which imho is fine. Should be good to go now.